File: src/test/regression_testing/test_files/Python/NPD/bad/test01.py
def test1_get_user_data():
    user = None
    return user


def test1_process_user():
    user_obj = test1_get_user_data()
    return user_obj.get("name")


def test1_main():
    result = test1_process_user()
    print(f"User name: {result}")


test1_main()

File: src/test/regression_testing/test_files/Go/NPD/bad/test01.go
package main

import "fmt"

type test1_MyStruct struct {
    Name string
}

func (m *test1_MyStruct) test1_PrintName() {
    fmt.Println(m.Name) 
}

func test1_main() {
    var s *test1_MyStruct 
    s.PrintName()    
}

func main() {
	test1_main()
}